integral_view/forms/zfs_forms.py

```diff
@@ -20,6 +20,11 @@ def clean(self):
                 ["The name cannot start with a number."])
             del cd["name"]
 
```

Comment:
There is no DeleteSnapshots which is referred to in the views. Also, we shd call it DeleteSnapshotsForm just to keep the naming convention